### PR TITLE
Add Autoscaler Example

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: autoscaler/autoscaler/autoscaler.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        time.sleep(short_sleep)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Detects text in the images in the given directory.')
+    parser.add_argument(
+        '--high_cpu_threshold',
+        help='If Bigtable CPU usages is above this threshold, scale up',
+        default=0.6)
+    parser.add_argument(
```
Comment (rameshdharan) [inline autoscaler/autoscaler/autoscaler.py]: I think you wanted "--low_cpu_threshold" for this one.

File: autoscaler/autoscaler/autoscaler.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        elif cluster_cpu < low_cpu_threshold:
+            scale_bigtable(False)
+            time.sleep(short_sleep)
+        else:
+            print('CPU within threshold, sleeping.')
+        time.sleep(short_sleep)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Detects text in the images in the given directory.')
```
Comment (rameshdharan) [inline autoscaler/autoscaler/autoscaler.py]: Copy/paste error?

**Comment by waprin** [comment]
Moving this PR here:

https://github.com/GoogleCloudPlatform/python-docs-samples/pull/943

**Comment by rameshdharan** [comment]
Agreed that python-docs-samples is a fine place for it, but please update the README.md in this project (cloud-bigtable-examples) as well once it has landed. There's a Python section here already that links to other examples in the python-docs-samples project.

**Pull request closed without merging**